package-system/pyside2/Findpyside2.cmake.Linux

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+     
+set(MY_NAME "pyside2")
+set(TARGET_WITH_NAMESPACE "3rdParty::${MY_NAME}")
+if (TARGET ${TARGET_WITH_NAMESPACE})
+    return()
+endif()
+
+add_library(${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
+
+set(PYSIDE_BASE_PATH ${CMAKE_CURRENT_LIST_DIR}/pyside2)
+
+ly_pip_install_local_package_editable(${PYSIDE_BASE_PATH}/lib/python3.10/site-packages pyside2)
+
+set(${MY_NAME}_RUNTIME_DEPENDENCIES
+    ${PYSIDE_BASE_PATH}/lib/libpyside2.abi3.so.5.15.2.1
+    ${PYSIDE_BASE_PATH}/lib/libpyside2.abi3.so.5.15
+    ${PYSIDE_BASE_PATH}/lib/libpyside2.abi3.so
+    ${PYSIDE_BASE_PATH}/lib/libshiboken2.abi3.so.5.15.2.1
+    ${PYSIDE_BASE_PATH}/lib/libshiboken2.abi3.so.5.15
+    ${PYSIDE_BASE_PATH}/lib/libshiboken2.abi3.so
```

Comment:
Seems like we should make two SHARED_IMPORTED GLOBAL targets for libpyside2 and libshiboken2, that is then added as a dependency of the pyside2 library

Reply:
(per offline discussion), the pyside2 target is used to specify dependent files that need to be copied to the runtime bin folder of the editor, and it is not linked against any code in O3DE. The binary files are needed by python when pyside2 is invoked through python, and having it reside in the same folder as the executable and the qt shared libraries ensures that the pyside site-packages and binaries link to the local o3de-specific Qt